CMakeLists.txt

```diff
@@ -110,6 +110,10 @@ option(WITH_CHISEL "Include chisel implementation" OFF)
 
 option(CREATE_TEST_TARGETS "Enable make-targets for unit testing" ON)
 
+if(WITH_CHISEL)
```

Comment:
Am not sure whether we want something like this; cmake already warns that the configure option is not available, and i don't see the use case for it; moreover release notes for libs 0.16 will have the `breaking change` entry.

Reply:
I think it can be useful for the consumers, maybe I can change `WARNING` with `ERROR` so the compilation stops immediately. Otherwise I can remove it :)

Reply:
@leogr  wdyt?

Reply:
I agree with @FedeDP, we can remove this.